### PR TITLE
perf: use fast api for op_now

### PR DESCRIPTION
**Pull request by littledivy**
2x faster `performance.now()`

```
deno run --allow-hrtime cli/bench/op_now.js # Before
time 482 ms rate 20746887
time 462 ms rate 21645021
time 462 ms rate 21645021
time 462 ms rate 21645021
time 467 ms rate 21413276
^C
                                                                                                                                                                          
target/release/deno run --allow-hrtime cli/bench/op_now.js # After
time 287 ms rate 34843205
time 267 ms rate 37453183
time 266 ms rate 37593984
time 265 ms rate 37735849
time 266 ms rate 37593984
^C
```

### CODE REVIEW

File: ext/web/02_timers.js [hunk 1]
```diff
@@ -25,8 +26,15 @@
   const { reportException } = window.__bootstrap.event;
   const { assert } = window.__bootstrap.infra;
 
+  let opNowInitialized = false;
+  const hr = new Uint32Array(2);
```
Comment (ry) [inline ext/web/02_timers.js:30]: Can hr also be constructed lazily ?

**Review by ry — APPROVED** [review]
LGTM

File: ext/web/timers.rs [hunk 2]
```diff
@@ -23,29 +24,50 @@ pub trait TimersPermission {
...
   if !state.borrow_mut::<TP>().allow_hrtime() {
+    let reduced_time_precision = 2_000_000; // 2ms in nanoseconds
     subsec_nanos -= subsec_nanos % reduced_time_precision;
   }
 
-  (seconds * 1_000) as f64 + (subsec_nanos / 1_000_000.0)
+  // SAFETY: This is safe because we initialize NOW_BUF in op_now_set_buf, its a null pointer
+  // otherwise.
+  // op_now_set_buf guarantees that the buffer is 8 bytes long.
+  unsafe {
```
Comment (billywhizz) [inline ext/web/timers.rs:64]: is it possible to call `Deno.core.ops.op_now.fast()` from userland? if so, won't we segfault here?

Comment (littledivy) [reply ext/web/timers.rs:64]: NOW_BUF is null if not set. `if !NOW_BUF.is_null() {` This check makes sure we don't dereference it.

**Pull request merged**